src/server.py

```diff
@@ -742,45 +742,34 @@ def __player_released_cb(self, unused_player, sound_event_id, uuid_):
             sound_event_id=self.get_player(uuid_).sound_event_id,
             uuid=uuid_
         )
-        self._resume_last_bg_player(uuid_)
-        del self.players[uuid_]
-        if sound_event_id in self._uuid_by_event_id:
-            self._uuid_by_event_id[sound_event_id].remove(uuid_)
-            if len(self._uuid_by_event_id[sound_event_id]) == 0:
-                del self._uuid_by_event_id[sound_event_id]
-        for bus_name in self._refcount[uuid_]:
-            if bus_name in self._watcher_by_bus_name:
-                self._watcher_by_bus_name[bus_name].uuids.remove(uuid_)
-        del self._refcount[uuid_]
+        self.__free_registry(sound_event_id, uuid_)
         if not self.players:
             self._ensure_release_countdown()
         self.release()
 
-    def __player_error_cb(self, player, error, debug, sound_event_id,
+    def __player_error_cb(self, unused_player, error, debug, sound_event_id,
                           uuid_, connection, path, iface):
         # This method is only called when the player fails or when an
         # application ordered to stop the sound. In both cases this means to
         # delete the references to that sound UUID.
         data = (uuid_, error.message, error.domain, error.code, debug)
         vdata = GLib.Variant("(sssis)", data)
```

Comment:
You don't need to create this variant until later when you emit the signal, so you could move it after the `uuid_` condition check

Reply:
done